### PR TITLE
cicd: update stalebot

### PR DESCRIPTION
**Pull request by korikuzma**
* create reusable workflow (it might make sense to make other actions as reusable workflows)
* add stale actions for high/medium/low priority labels

### CODE REVIEW

**Review by wesleygoar — APPROVED** [review]

**Comment by jsstevenson** [comment]
Does this make `bug` also stale-exempt?

**Comment by korikuzma** [comment]
> @korikuzma Does this make `bug` also stale-exempt?

@jsstevenson Not for this repo. I just used the default label `stale-exempt`. When other repos use the reusable-stale.yaml, they can have a job that specifies exempt labels and that can include bug if needed. However, I'm kind of hesitant about marking too many issues as stale exempt because it might reduce the effectiveness of stalebot. Sometimes issues labeled as bug have been completed, but have not been closed. So I think stale-exempt labels to be used sparingly. Open to other people's thoughts on this.

**Comment by korikuzma** [comment]
Another comment is for repos that have many issues without priority labels, we might want something like:

```yaml
  stale-no-priority-labels:
    uses: ./.github/workflows/reusable-stale.yaml
    with:
      days-before-issue-stale: 90
      days-before-pr-stale: 7
      exempt-labels: stale-exempt,priority:low,priority:medium,priority:high
```

**Review by wesleygoar — DISMISSED** [review]

**Review by jsstevenson — DISMISSED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -5,24 +5,23 @@ on:
     - cron: "30 13 * * *"
```
Comment (jsstevenson) [inline .github/workflows/stale.yml:5]: what about `30 13 * * 1-5` so it only runs on weekdays?

Comment (korikuzma) [reply .github/workflows/stale.yml:5]: I like that

**Review by jsstevenson — APPROVED** [review]

**Review by wesleygoar — APPROVED** [review]

**Review by katiestahl — APPROVED** [review]

**Review by MichaelMcNCH — APPROVED** [review]

**Pull request merged**